Added example to use SaaS API URL in partition_via_api (#2512)
Unstructured-IO/unstructured: docs/source/apis/saas_api.rst

    curl -X 'POST' 'https://<REPLACE WITH THE URL IN THE EMAIL>'
         -H 'accept: application/json'
         -H 'Content-Type: multipart/form-data'

>          -H 'unstructured-api-key: <REPLACE WITH API KEY IN THE EMAIL>/general/v0/general'

Reviewer: ```suggestion
         -H 'unstructured-api-key: <REPLACE WITH API KEY IN THE EMAIL>'
```
the `/general/v0/general` should go with the POST url location (line 74) instead

         -F 'files=@sample-docs/family-day.eml'

.. image:: imgs/paid_api/sample_output.png
  :align: center
  :alt: Sample Output

Or, you can use `partition_via_api <https://unstructured-io.github.io/unstructured/core/partition.html#partition-via-api>`__ to access the SaaS API URL.

.. code:: python

  from unstructured.partition.api import partition_via_api

  filename = "example-docs/eml/fake-email.eml"

>   elements = partition_via_api(
>     filename=filename,

Reviewer: ```suggestion
  elements = partition_via_api(
    filename=filename,
    api_key=<<REPLACE WITH YOUR API KEY>>,
```
might be helpful to include both key and url in the example for clarity on what to do with the credentials they receive

>     api_url="http://<<REPLACE WITH YOUR API KEY>>/general/v0/general"

Reviewer: ```suggestion
    api_url="https://<<REPLACE WITH YOUR SERVER URL>>/general/v0/general"
```
